loops-carlos-paula.js

```diff
@@ -0,0 +1,49 @@
+// // Write a for loop that logs each number from 1 - 20.
+// //
+// // for(let i=1; i <21; i++){
+// //   console.log(i)
+// // }
+// // Write a for loop that logs the result of each number from 1 - 20 tripled.
+// // for(let i=1; i<21; i++){
+// //   console.log([i]*3)
+// // }
+// // Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+// for(let i=1; i <21; i++){
+//   if([i] % 2 !== 0){
+//     console.log("ODD")
+//   } else {console.log(i)}
+// }
+
+
+var nums = [3, 57, -9, 20, 67]
+
+
+// Create a loop that will log the highest number from the array. Expected output --> 67
+
+// let high = nums [0]
+
+// for (let i=0; i<nums.length; i++){
+//  if (nums[i]>high) {
+//    high=nums[i]
+//   }
+// }
+  
+// console.log(high)
+
+
+// let low = nums [0]
+
+// for (let i=0; i<nums.length; i++){
+//  if (nums[i]<low) {
+//    low=nums[i]
+//   }
+// }
+// console.log(low)
+
+
+
+for(let i=1; i <nums.length; i++){
+  if([i] % 2 ){
```

Comment:
This challenge question was looking for the remainder of each number divided by two, so console.log using the modulo operator would be suffice in this for loop. Lines 46 - 48 can be refactored using
`console.log(nums[i] % 2)`